nima_core/__init__.py

```
@@ -58,12 +92,14 @@
     """CLI entry point — runs setup wizard."""
     import subprocess, sys
+
     # Find and run install.sh from package directory
     import os
+
     pkg_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    install_script = os.path.join(pkg_dir, 'install.sh')
+    install_script = os.path.join(pkg_dir, "install.sh")
     if os.path.exists(install_script):
-        subprocess.run(['bash', install_script], cwd=pkg_dir)
+        subprocess.run(["bash", install_script], cwd=pkg_dir)
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)